dev_guide/deployments.adoc

```diff
@@ -191,6 +191,52 @@ To see what the rollback would look like without performing the rollback:
 $ oc rollback <deployment_config> --dry-run
 ----
 
+[[executing-commands-inside-a-container-deployments]]
+== Executing Commands Inside a Container
+
+You can add a command to a container, which it runs every time the container
```

Comment:
it's really more than adding a command to the container.  The command field replaces the image's entrypoint, so whatever startup behavior the image would normally have, is superceded by this value.

Similarly the "args" field will be passed as arguments to the ENTRYPOINT if the image has one (or the command if that field is being specified).  So again, changes the behavior of the container entirely, replacing the normal behavior.  If the image has no ENTRYPOINT and the "command" field is not being specified, then the "args" will be invoked directly, i believe (rather than being passed as arguments to the entrypoint/command value).